src/analyses/extractPthread.ml

```diff
@@ -171,9 +167,8 @@ module Tbls = struct
     let extend k v = Hashtbl.modify_def Set.empty k (Set.add v) table
 
     let get_fun_for_tid v =
-      Hashtbl.keys table
-      |> List.of_enum
-      |> List.find (fun k ->
+      Hashtbl.to_seq_keys table
+      |> Seq.find (fun k ->
           Option.get @@ Hashtbl.find table k |> Set.exists (( = ) v))
```

Comment:
This also looks like some `get_key` thing. With `to_seq` one could avoid the `find` at least.